Bulk update: Adding dois and month entries

stschiff opened the pull request:
### PR Checklist for modifying one or multiple existing packages

- [x] The changes maintain the structural integrity of the affected packages.
- [x] The checksums of the modified files in the respective `POSEIDON.yml` files were adjusted properly.
- [x] Every file in the submission is correctly referenced in the relevant `POSEIDON.yml` files and there are no additional, supplementary files in the submission that are not documented there.

***

- [x] The `packageVersion` numbers of the affected packages were increased in their `POSEIDON.yml` files.
- [x] The changes in the `packageVersion` followed the Poseidon [Package versioning policy](https://github.com/poseidon-framework/poseidon-schema?tab=readme-ov-file#package-versioning).
- [x] The changes were documented in the respective `CHANGELOG` files. If no `CHANGELOG` files existed previously it was added here.
- [x] The `lastModified` fields of the affected `POSEIDON.yml` files were updated.
- [x] The `contributor` fields were updated with `name`, `email` and `orcid` of the relevant, new contributors.
- [x] The `.janno` and the `.ssf` files are not fully quoted, so they only use single- or double quotes (`"..."`, `'...'`) to enclose text fields where it is strictly necessary (i.e. their entry includes a TAB).

***

- [x] All affected packages pass a validation with `trident validate --fullGeno`.

***

- [x] Large genotype data files are properly tracked with Git LFS and not directly pushed to the repository. For an instruction on how to set up Git LFS please look [here](https://www.poseidon-adna.org/#/archive_submission_guide?id=submitting-the-package). If you accidentally pushed the files the wrong way you can fix it with `git lfs migrate import --no-rewrite path/to/file.bed` (see [here](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-migrate.adoc#import-without-rewriting-history)).

stschiff commented:
This is a massive patch update to dozens of package bib-entries whose doi was missing. I've started analysing the bib-data recently and needed these updates. I hope it's tolerable to check this PR. It's really only added DOIs.

In two packages, I also updated the Janno, as the bibkey contained annoying typos of the first author's name, which I also fixed.

nevrome commented on 2015_1000Genomes_1240K_haploid_pulldown/LITERATURE.bib:
```diff
@@ -2,6 +2,7 @@ @article{1KGPhase3
   title         = "A global reference for human genetic variation",
   author        = "{1000 Genomes Project Consortium} and Auton, Adam and Brooks, Lisa D and Durbin, Richard M and Garrison, Erik P and Kang, Hyun Min and Korbel, Jan O and Marchini, Jonathan L and McCarthy, Shane and McVean, Gil A and Abecasis, Gon{\c c}alo R",
   journal       = "Nature",
+  doi           = dx.doi.org/10.1038/nature15393,
```
I think quotes or curly bracket are necessary. The respective POSEIDON.yml also needs a checksum update.

nevrome commented on 2018_Ebenesersdottir_Iceland/LITERATURE.bib:
```diff
@@ -2,6 +2,7 @@ @article{EbenesersdottirScience2018
   title         = "Ancient genomes from Iceland reveal the making of a human population",
   author        = "Ebenesersd{\'o}ttir, S Sunna and Sandoval-Velasco, Marcela and Gunnarsd{\'o}ttir, Ellen D and Jagadeesan, Anuradha and Gu{\dh}mundsd{\'o}ttir, Vald{\'\i}s B and Thordard{\'o}ttir, El{\'\i}sabet L and Einarsd{\'o}ttir, Margr{\'e}t S and Moore, Kristjan H S and Sigur{\dh}sson, {\'A}sgeir and Magn{\'u}sd{\'o}ttir, Droplaug N and J{\'o}nsson, H{\'a}kon and Snorrad{\'o}ttir, Steinun ...
   journal       = "Science",
+  doi           = "DOI: 10.1126/science.aar2625",
```
`DOI: ` should be removed.

nevrome commented on 2019_Feldman_Philistines/LITERATURE.bib:
```diff
@@ -2,6 +2,7 @@ @article{FeldmanScienceAdvances2019
   title         = "Ancient {DNA} sheds light on the genetic origins of early Iron Age Philistines",
   author        = "Feldman, Michal and Master, Daniel M and Bianco, Raffaela A and Burri, Marta and Stockhammer, Philipp W and Mittnik, Alissa and Aja, Adam J and Jeong, Choongwon and Krause, Johannes",
   journal       = "Sci Adv",
+  doi           = "DOI: 10.1126/sciadv.aax0061",
```
Remove `DOI: `

nevrome commented on 2019_GonzalesFortes_Iberia/LITERATURE.bib:
```diff
@@ -2,6 +2,7 @@ @article{GonzalesFortesProcRoyalSocB2019
   title         = "A western route of prehistoric human migration from Africa into the Iberian Peninsula",
   author        = "Gonz{\'a}lez-Fortes, G and Tassi, F and Trucchi, E and Henneberger, K and Paijmans, J L A and D{\'\i}ez-Del-Molino, D and Schroeder, H and Susca, R R and Barroso-Ru{\'\i}z, C and Bermudez, F J and Barroso-Medina, C and Bettencourt, A M S and Sampaio, H A and Grandal-d'Anglade, A and Salas, A and de Lombera-Hermida, A and Fabregas Valcarce, R and Vaquero, M and Alonso, S and Loz ...
   journal       = "Proc. Biol. Sci.",
+  doi           = "https://doi.org/10.1098/rspb.2018.2288",
```
The others so far didn't include `https://doi.org/`

nevrome commented on 2019_Schroeder_Poland/LITERATURE.bib:
```diff
@@ -2,6 +2,7 @@ @article{SchroederPNAS2019
   title         = "Unraveling ancestry, kinship, and violence in a Late Neolithic mass grave",
   author        = "Schroeder, Hannes and Margaryan, Ashot and Szmyt, Marzena and Theulot, Bertrand and W{\l}odarczak, Piotr and Rasmussen, Simon and Gopalakrishnan, Shyam and Szczepanek, Anita and Konopka, Tomasz and Jensen, Theis Z T and Witkowska, Barbara and Wilk, Stanis{\l}aw and Przyby{\l}a, Marcin M and Pospieszny, {\L}ukasz and Sj{\"o}gren, Karl-G{\"o}ran and Belka, Zdzislaw and Olsen, Jes ...
   journal       = "Proc. Natl. Acad. Sci. U. S. A.",
+  doi           = "https://doi.org/10.1073/pnas.1820210116",
```
Again with `https://doi.org/`

nevrome commented on 2019_Villalba_Iberia/POSEIDON.yml:
```diff
@@ -1,12 +1,12 @@
-description: Survival of Late Pleistocene Hunter-Gatherer Ancestry in the Iberian. Originally AADR v42.4.
-  Peninsula.
+description: Survival of Late Pleistocene Hunter-Gatherer Ancestry in the Iberian.
+  Originally AADR v42.4. Peninsula.
```
This would be a chance to fix the description:

> Survival of Late Pleistocene Hunter-Gatherer Ancestry in the Iberian Peninsula. Originally AADR v42.4.

stschiff commented:
Oh boy, sorry for overlooking these obvious problems. Will fix them ASAP.

stschiff commented:
OK, fixed it.

stschiff commented:
Please don't review this yet, I'd like to add more updates.

stschiff commented:
Ooookay, I've added more changes. Now I've also updated the "month" field in the bib-files. A total of 67 packages have been updated. I think a review should be reasonably quick if you pull this branch and then run 

```git diff master```

Sorry for the massive update, but I thought perhaps it's better to just have one update in the versions instead of two separate pull requests with two version updates. Not sure whether we have a recommendation for that. 

Thanks for going through it!
